frontend/src/lib/notification.tsx

```diff
@@ -0,0 +1,36 @@
+import { setUINotifications } from '../redux/actions/actions';
+import store from '../redux/stores/store';
+
+enum NotificationType {
+  DIALOG = 1 << 0,
+  SNACKBAR = 1 << 1,
+  ALL = DIALOG | SNACKBAR,
+}
+
+export class Notification {
+  static NotificationType = NotificationType;
+  type: NotificationType = NotificationType.DIALOG;
```

Comment:
Could this class be documented? It would be great if all interfaces and functions could be documented... but these in particular are exposed to the public.

Reply:
I have created a mention of the class in the plugin doc, is that what you meant @illume ??

Reply:
I'm meant documented with tsdoc strings.